CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [1.3.0] 2024-03-12
 
 ### Fixed
+- Display selected item in playback menu [RGOeX-25733]
```

Comment:
@Lunyachek this is not the v1.3.0 changes

Please move it to "Unreleased"

Also, it would be better to rebase in the future instead of merging the master branch (commit history will be a bit cleaner)